### PR TITLE
Add All Licenses in Source Repo, Refresh Recipe

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge/core - I have refreshed and fixed this feedstock to add all the licenses and refreshed the current version.
Can I be added as a maintainer - I am happy to help keep this feedstock updated.
Thank you and Regards

File: recipe/meta.yaml [hunk 1]
```diff
@@ -68,9 +67,141 @@ about:
...
   summary: Question and Answer forum, like StackOverflow, written in python and Django
   doc_url: https://pythonhosted.org/askbot/
-  license: BSD-3-Clause
-  license_file: LICENSE
+  license: MIT AND LGPL-3.0-or-later AND BSD-3-Clause AND GPL-3.0-or-later AND Apache-2.0 AND BSD-2-Clause
+  license_file:
+    - LICENSE
+    - COPYING
+    - askbot/conf/license.py
+    - askbot/media/node_modules/strip-json-comments/license
```
Comment (xhochy) [inline recipe/meta.yaml]: For maintainability, have a look at other uses of `pnpm-licenses` in other feedstocks. This aggregates all licenses into one file.

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipe/meta.yaml]: @xhochy - this is interesting, but might be beyond my capabilities - could you help here to make this use pnpm-licenses to aggregate the licenses..

**Comment by rxm7706** [comment]
@xhochy thank you. This is helpful to see and learn from. Regards.

**Pull request merged**